compiler/rustc_middle/src/ty/sty.rs

```diff
@@ -1505,6 +1516,23 @@ impl<'tcx> Region<'tcx> {
         *self.0.0
     }
 
+    pub fn get_name(self) -> Option<Symbol> {
+        if self.has_name() {
+            let name = match *self {
+                ty::ReEarlyBound(ebr) => Some(ebr.name),
+                ty::ReLateBound(_, br) => br.kind.get_name(),
+                ty::ReFree(fr) => fr.bound_region.get_name(),
+                ty::ReStatic => Some(Symbol::intern("static")),
```

Comment:
```suggestion
                ty::ReStatic => Some(Symbol::intern("'static")),
```
i think?

Reply:
though probably add that as a predefined symbol to `sym`.

Reply:
```suggestion
                ty::ReStatic => Some(sym::StaticLifetime),
```